home/models.py

```diff
@@ -94,6 +94,10 @@ def get_context(self, request):
     def get_image_urls(self):
         return self._get_stream_data_image_urls(self.home_featured_content.raw_data)
 
+    @property
+    def get_media_urls(self):
+        return self._get_stream_data_media_urls(self.home_featured_content.raw_data)
```

Comment:
I don't understand why this implemented in this way because `home_featured_content` doesn't have a 'media' block - this will return an empty list every time.